server/TypeDBServer.java

```diff
@@ -155,7 +155,7 @@ private void configureTracing() {
     protected io.grpc.Server rpcServer() {
         assert Executors.isInitialised();
 
-        typeDBService = new TypeDBService(databaseMgr);
+        typeDBService = new TypeDBService(config.server().address(), databaseMgr);
```

Comment:
`TypeDBService` now needs to know its own address in order to be able to meaningfully reply to the `serversAll()` request and to provide the database replica address.